ui/perfherder/compare/CompareSubtestsView.jsx

```diff
@@ -133,6 +133,9 @@ class CompareSubtestsView extends React.PureComponent {
       const newResults = newResultsMap.get(testName);
 
       const cmap = getCounterMap(testName, oldResults, newResults);
+      if (!newResults || !oldResults) {
```

Comment:
I think this check is accomplished by the next check (`if (cmap.isEmpty)`), making this redundant.